pkg/filter/stream/flowcontrol/stream_filter_factory.go

```diff
@@ -73,6 +101,9 @@ func createRpcFlowControlFilterFactory(conf map[string]interface{}) (api.StreamF
 		log.DefaultLogger.Errorf("update rules failed")
 		return nil, err
 	}
+	initOnce.Do(func() {
+		initSentinel(flowControlCfg.AppName, flowControlCfg.LogPath)
```

Comment:
Maybe TODO
support dynamicly update config for initSentinel.

Reply:
done

Reply:
> Maybe TODO
> 
> support dynamicly update config for initSentinel.

done